components/admin_console/team_channel_settings/team/details/team_details.tsx

```diff
@@ -159,12 +159,12 @@ export default class TeamDetails extends React.PureComponent<Props, State> {
                 browserHistory.push('/admin_console/user_management/teams');
             }
             return;
-        } else if (this.teamToBeRestored() && this.state.serverError === null) {
+        } else if (this.teamToBeRestored() && this.state.serverError == null) {
```

Comment:
`===` is converted to `==` as I changed `serverError` from `null` to `undefined`. Same for other lines below

Reply:
1/5, but I'd say we should just check for falsy here since the object is never actually going to be null.
```suggestion
        } else if (this.teamToBeRestored() && !this.state.serverError) {
```